libs/eo/metering-points/shell/src/lib/eo-metering-points-shell.component.ts

```diff
@@ -24,14 +27,34 @@ import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
       :host {
         display: block;
       }
+
+      .grey-color {
+        color: var(--watt-color-neutral-grey-900);
+      }
     `,
   ],
-  template: `<p>You do not have any metering points.</p>`,
+  template: `<ng-container *rxLet="meteringPoints$ as meteringPoints">
+    <p class="grey-color" *ngIf="meteringPoints.length < 1">
```

Comment:
suggestion: In this case we can type check if length === 0, IF we know that our API will always return an array, just with no values? If the API does not return an empty array, then I think we should suggest it or at least talk about it so that we either have "null" or empty arrays on null results across the different endpoint in the future.

```suggestion
    <p class="grey-color" *ngIf="meteringPoints.length === 0">
```

Reply:
Fixed :)